modules/gdscript/gdscript.cpp

```diff
@@ -111,22 +111,22 @@ GDScriptInstance *GDScript::_create_instance(const Variant **p_args, int p_argco
 	GDScriptLanguage::singleton->lock->unlock();
 #endif
 
```

Comment:
IMO this can simplified with
`if (p_argcount < 0) return instance;`

Reply:
I guess the reason why I haven't done this is simply because of the below clean up procedures, in some cases short-cutting with `return` statement is error-prone.

Reply:
Indeed this makes less diff changes. 🙂